### DISCUSSION TITLE
Add possibility to pass list of files/directories for scan

### DISCUSSION THREAD

**Discussion by bober2000** [aquasecurity/trivy#6460] [Ideas]
### Description

While implementing trivy scan using GitHub actions - I wan't to check only changed files/directories
But now I'm getting 'FATAL multiple targets cannot be specified' when trying to do this

### Target

Filesystem

### Scanner

Misconfiguration

**Comment by itaysk** [comment]
Duplicate of https://github.com/aquasecurity/trivy/discussions/4203
